tests/js/spec/views/organizationIntegrations/sentryAppDetailedView.spec.jsx

```diff
@@ -0,0 +1,269 @@
+      expect(wrapper.find('Name').props().children).toEqual('ClickUp');
+      expect(wrapper.find('IntegrationStatus').props().status).toEqual('Not Installed');
+    });
+    it('shows the Accept & Install button', async function() {
+      expect(wrapper.find('InstallButton').props().disabled).toEqual(false);
+      expect(wrapper.find('InstallButton').props().children).toEqual('Accept & Install');
+    });
+
+    it('onClick: installs app', async function() {
+      wrapper.find('InstallButton').simulate('click');
+      await tick();
+      wrapper.update();
+      expect(wrapper.find('IntegrationStatus').props().status).toEqual('Installed');
+    });
+  });
+
+  describe('Internal Sentry App', function() {
+    beforeEach(() => {
+      Client.clearMockResponses();
+
+      mockResponse([
+        [
+          '/sentry-apps/my-headband-washer-289499/',
+          {
+            status: 'internal',
+            scopes: [
+              'project:read',
+              'team:read',
+              'team:write',
+              'project:releases',
+              'event:read',
+              'org:read',
+              'member:read',
+              'member:write',
+            ],
+            isAlertable: false,
+            clientSecret:
+              '8f47dcef40f7486f9bacfeca257022e092a483add7cf4d619993b9ace9775a79',
+            overview: null,
+            verifyInstall: false,
+            owner: {id: 1, slug: 'sentry'},
+            slug: 'my-headband-washer-289499',
+            name: 'My Headband Washer',
+            uuid: 'a806ab10-9608-4a4f-8dd9-ca6d6c09f9f5',
+            author: 'Sentry',
+            webhookUrl: 'https://myheadbandwasher.com',
+            clientId: 'a6d35972d4164ef18845b1e2ca954fe70ac196e0b20d4d1e8760a38772cf6f1c',
+            redirectUrl: null,
+            allowedOrigins: [],
+            events: [],
+            schema: {},
+          },
+        ],
+        [
+          '/sentry-apps/my-headband-washer-289499/features/',
+          [
+            {
+              featureGate: 'integrations-api',
+              description:
+                'My Headband Washer can **utilize the Sentry API** to pull data or update resources in Sentry (with permissions granted, of course).',
+            },
+          ],
+        ],
+        [`/organizations/${org.slug}/sentry-app-installations/`, []],
+      ]);
+
+      wrapper = mountWithTheme(
+        <SentryAppDetailedView
+          params={{integrationSlug: 'my-headband-washer-289499', orgId: org.slug}}
+          location={{query: {}}}
+          router={router}
+        />,
+        routerContext
+      );
+
+      mockRouterPush(wrapper, router);
+    });
+    it('should get redirected to Developer Settings', () => {
+      expect(router.push).toHaveBeenLastCalledWith(
+        `/settings/${org.slug}/developer-settings/my-headband-washer-289499/`
+      );
+    });
+  });
+
+  describe('Unpublished Sentry App', function() {
```

Comment:
@NisanthanNanthakumar I don't think these tests are adding any value since unpublished and published apps are treated exactly the same on the detail view.